src/openms/include/OpenMS/FORMAT/PepXMLFile.h

```diff
@@ -149,6 +149,12 @@ namespace OpenMS
     */
     void matchModification_(const double mass, const String& origin, String& modification_description);
 
+    /**
+        @brief create a TPP compatible string of the modified sequence using bracket notation
+        i.e.: n[35]RQLNK[162]LQHK[162]GEA
+    */  
+    static String modifiedSequenceToBracketString_(const AASequence & seq);
```

Comment:
I think we do parse it. In aasequence maybe?

sent from my smartphone -- you may keep any typos you find.
On Aug 5, 2016 12:15 PM, "Timo Sachsenberg" <notifications@github.com>
wrote:

> In src/openms/include/OpenMS/FORMAT/PepXMLFile.h
> <https://github.com/OpenMS/OpenMS/pull/1933#discussion_r73745504>:
>
> > @@ -149,6 +149,12 @@ namespace OpenMS
> >      */
> >      void matchModification_(const double mass, const String& origin, String& modification_description);
> >
> > +    /**
> > +        @brief create a TPP compatible string of the modified sequence using bracket notation
> > +        i.e.: n[35]RQLNK[162]LQHK[162]GEA
> > +    */
> > +    static String modifiedSequenceToBracketString_(const AASequence & seq);
>
> I think this is never parsed by us from pepXML files and not required to
> determine the mod state but I can check this again. Given that there is no
> real standardized format for this string (at least that is how I understood
> it) I would only use it for writing out pepXML and not expose it to the
> API.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenMS/OpenMS/pull/1933/files/60d1854d6776d7a431c9ffdaadde3576bfca9db8..23c9880fd1e1b6a59b9eebfcfb5e01f40688cf7e#r73745504>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAcUjkQleQwtxOGoybMk8GpacA0JYrzzks5qc4u8gaJpZM4ID9wv>
> .
>